Src/Couchbase.Linq.NetStandard/UnixMillisecondsDateTimeOffset.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
```

Comment:
Instead of repeating this class for NetStandard, it should just be added as a Link to the file in the other project.  This must be done by manually editing the csproj file in a text editor.